source/supported-hardware/cml-rvp.rst

```diff
@@ -0,0 +1,228 @@
+
+
+Stitching
+^^^^^^^^^^
+
+1. Gather |CML| IFWI firmware image
+
+  Users can either download the full IFWI image if the IFWI image release is available, or read the exising IFWI image on the board using SPI programmer.
+  This image contains additional firmware ingredients that are required to boot on |CML|.
+
+.. note::
+  ``StitchLoader.py`` currently only supports stitching with boot guard feature **disabled**.
+  To stitch with Boot Guard enabled, please use ``StitchIfwi.py``.
+
+
+2. Stitch |SPN| images into downloaded BIOS image::
+
+    for PCH H and PCH H420E:
+    python Platform/CometlakeBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/cml/SlimBootloader.bin -o <SBL_IFWI_IMAGE_NAME>
+
+    for PCH V:
+    python Platform/CometlakevBoardPkg/Script/StitchLoader.py -i <BIOS_IMAGE_NAME> -s Outputs/cmlv/SlimBootloader.bin -o <SBL_IFWI_IMAGE_NAME>
+
+    where -i = Input file, -o = Output file.
+
+For example, stitching |SPN| IFWI image ``sbl_cml_ifwi.bin`` from |CMLH| firmware images downloaded::
+
+    python Platform/CometlakeBoardPkg/Script/StitchLoader.py -i xxxx.bin -s Outputs/cml/SlimBootloader.bin -o sbl_cml_ifwi.bin
+
+For more details on stitch tool, see :ref:`stitch-tool` on how to stitch the IFWI image with |SPN|.
+
+
+Flashing
+^^^^^^^^^
+
+Flash the generated ``sbl_cml_ifwi.bin`` or ``sbl_cmlv_ifwi.bin`` to the target board using DediProg SF100 or SF600 programmer.
+
+
+.. note:: Please check the alignment/polarity when connecting Dediprog to the board. Please power off the board before connecting the Dediprog.
+.. note:: Please set the dediprog  to flash the ifwi binary from offset 0
+
+.. note:: The connector labelled **J7G1** on the target board is for DediProg.
+.. note:: Please disconnect Deidprog before powering up the board again.
+
+
+Slimbootloader binary for capsule image
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Please refer to the build command above or :ref:`build-tool`.
+
+slimbootloader binary generated in ``outputs\cml\slimbootloader.bin`` or ``outputs\cmlv\slimbootloader.bin`` can be used for generating capsule image
+
+
+Capsule image for |CML|
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To generate capsule image for |CML| platform::
+
+    for PCH H and PCH H420E:
+    python ./BootloaderCorePkg/Tools/GenCapsuleFirmware.py -p BIOS Outputs/cml/SlimBootloader.bin -k <Keys> -o FwuImage.bin
+
+    for PCH V:
+    python ./BootloaderCorePkg/Tools/GenCapsuleFirmware.py -p BIOS Outputs/cmlv/SlimBootloader.bin -k <Keys> -o FwuImage.bin
+
+For more details on generating capsule image, please refer :ref:`generate-capsule`.
+
+
+Triggering Firmware Update
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sample implementation of trigerring firmware update is explained below
+
+|SPN| for |CML| uses BIT16 of PMC I/O register (Over-Clocking WDT Control (OC_WDT_CTL) - Offset 54h) to trigger firmware update. When BIT16 is set, |SPN| will set the boot mode to FLASH_UPDATE.

```

Comment:
Let us remove the details on which register is used for triggering.  Also remove the details on the C files and function name IsFirmwareUpdate().   The page should provide very high level info.  
Instead, I think we just mention:
Please refere to https://slimbootloader.github.io/security/firmware-update.html on how to trigger firmware update flow.  Below is an example:

Reply:
Done the changes based on the comment :)